src/logging.cc

```diff
@@ -628,7 +628,7 @@ inline void LogDestination::RemoveLogSink(LogSink *destination) {
   MutexLock l(&sink_mutex_);
   // This doesn't keep the sinks in order, but who cares?
   if (sinks_) {
-    for (int i = sinks_->size() - 1; i >= 0; i--) {
+    for (int i = (int)sinks_->size() - 1; i >= 0; i--) {
```

Comment:
ssize_t makes no sense since the comparison >=0 would be useless. Using ptrdiff_t.